services/api/utils.go

```diff
@@ -203,3 +244,26 @@ func getHeaderContentType(header http.Header) (mediatype string, params map[stri
 
 	return contentType, params, nil
 }
+
+func getSlotFromBuilderJSONPayload(input []byte) (uint64, error) {
+	slot := gjson.Get(string(input), "message.slot")
```

Comment:
we already include a fast json parser: https://github.com/buger/jsonparser

maybe use that one instead of adding a second fast one? not feeling strongly about it, we could also consider switching to gjson (it has more stars).

Reply:
i would kinda lean to using gjson since it also seems more well maintained. last commit to jsonparser is 3 years ago